flink-contrib/src/test/java/org/apache/flink/contrib/SimpleTweetInputFormatTest.java

```diff
@@ -0,0 +1,62 @@
+package org.apache.flink.contrib;
+
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.contrib.tweetinputformat.io.SimpleTweetInputFormat;
+import org.apache.flink.contrib.tweetinputformat.model.tweet.Tweet;
+import org.apache.flink.contrib.tweetinputformat.model.tweet.entities.HashTags;
+import org.apache.flink.core.fs.FileInputSplit;
+import org.apache.flink.core.fs.Path;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+
+
+public class SimpleTweetInputFormatTest {
+
+    private Tweet tweet;
+
+    private SimpleTweetInputFormat simpleTweetInputFormat;
+
+    private FileInputSplit fileInputSplit;
+
+    protected Configuration config;
+
+    protected File tempFile;
+
+
+    @Before
+    public void testSetUp() {
+
+
+        simpleTweetInputFormat = new SimpleTweetInputFormat();
+
+        File jsonFile = new  File("../flink-contrib/src/main/resources/HashTagTweetSample.json");
+
+        fileInputSplit = new FileInputSplit(0, new Path(jsonFile.getPath()), 0, jsonFile.length(), new String[] {"localhost"});
+    }
+
+    @Test
+    public void testTweetInput() throws Exception {
+
+
+        simpleTweetInputFormat.open(fileInputSplit);
+        while (!simpleTweetInputFormat.reachedEnd()) {
```

Comment:
Changed, I have tried to use "Switch" instead of multiple "if-else" because it more efficient, but the compiler of flink uses Java 1.6, and it can not accept "String" in switches.